user_update.html

```diff
@@ -57,9 +57,23 @@ <h3 class="section-header meta margin-bottom-large">{{ user.name }}</h3>
 					{% case 'email' %}
 					{% if user.stats.donated %}
 						<p class="donor-email-label">We noticed you have previously donated. To update your email address ({{ field.value }}), please contact us at <a href="mailto:donations@350.org">donations@350.org</a> so we can make sure your donation records are moved to your new address.</p>
+						<input type="hidden" name="email" value="{{ field.value }}" maxlength="255" id="id_email" readonly>

```

Comment:
Is this missing the updates made in the event create pages here or are those already in master? https://github.com/350org/ak_intl_v3/pull/470/files

Reply:
those were made in other commits (not authored by me) which I didn't want to include in this PR. it doesn't look like they're in master, so I'm not really sure where they came from 🤔

Reply:
oh nvm, they are included in master, they're just quite old: https://github.com/350org/ak_intl_v3/commit/38cce1b6a654a59eafc8268289e6251567a68784

Reply:
Ah got it, makes sense!